draft-ietf-oauth-transaction-tokens.md

```diff
@@ -740,6 +740,7 @@ The authors would like to thank the contributors and the OAuth working group mem
 # Document History
 {: numbered="false"}
 [[ To be removed from final specification ]]
+* Replace StringOrURI with string [Relace StringOrURI with String](https://github.com/oauth-wg/oauth-transaction-tokens/issues/195)
```

Comment:
need (i think) another line or this renders funny but also shouldn't this go under "Since Draft 06"?

<img width="660" height="190" alt="Screenshot 2025-09-17 at 2 58 46 PM" src="https://github.com/user-attachments/assets/01c6165e-f286-40c4-b4c1-d6bfbc928301" />